src/python/tests/core/crash_analysis/stack_parsing/stack_analyzer_test.py

```diff
@@ -2950,3 +2950,16 @@ def test_rust_oom(self):
     self._validate_get_crash_data(data, expected_type, expected_address,
                                   expected_state, expected_stacktrace,
                                   expected_security_flag)
+
+  def test_linux_kernel_library_libfuzzer(self):
+    """Test Linux Kernel Library fuzzed with libfuzzer ."""
+    data = self._read_test_data('lkl_libfuzzer.txt')
```

Comment:
We shouldnt call function here in test code, this should be an "lkl_libfuzzer_stripped.txt" file and then we read it here for raw contents.